pkg/cpi/mockingUtils.go

```diff
@@ -41,3 +56,27 @@ func GetCPIFunctionMockResponse(functionName, testType string) (*http.Response,
 		return &res, errors.New("Service not Found")
 	}
 }
+
+//GetEmptyHTTPResponseBody -Empty http respose body
+func GetEmptyHTTPResponseBody() (*http.Response, error) {
+	res := http.Response{
+		StatusCode: 202,
+		Body:       ioutil.NopCloser(bytes.NewReader([]byte(``))),
+	}
+	return &res, nil
+}
+
+//GetNegativeCaseHTTPResponseBody -Negative case http respose body
+func GetNegativeCaseHTTPResponseBody() (*http.Response, error) {
+	res := http.Response{
+		StatusCode: 400,
+		Body: ioutil.NopCloser(bytes.NewReader([]byte(`{
+					"code": "Bad Request",
+					"message": {
+					"@lang": "en",
+					"#text": "Wrong body format for the expected parameter value"
+					}
+				}`))),
+	}
+	return &res, nil
```

Comment:
Why return an error if it is always `nil`?

Reply:
this function cater to positive test case where we don't worry about error

Reply:
There is no case in the `GetNegativeCaseHTTPResponseBody` returns an error. So it is not needed in the return values. I would suggest adding the nil-error to the return statement where `GetNegativeCaseHTTPResponseBody` is used.